tools/c7n_mailer/README.md

```diff
@@ -502,7 +550,7 @@ The following extra jinja filters are available:
 
 Clone the repository:
 ```
-$ git clone https://github.com/cloud-custodian/cloud-custodian
```

Comment:
the urls shouldn't be changed, this looks like a bad merge of trunk into a branch wrt to the readme changes.

Reply:
The README.md has been corrected in the latest commit. I actually wrote the README.md last month using the older version before the doc sprint. I just commited the latest README.md but with the gcp content now.